packages/api-client/src/TamanuApi.js

```
@@ -289,4 +365,103 @@
+          // | 4 bytes | 4 bytes | $length$ bytes |
+          // +---------+---------+----------------+
+          //
+          // This framing makes it cheap to verify that all the data is here,
+          // and also doesn't *require* us to parse any of the message data.
+
+          const kind = partial.readUInt32BE(0);
+          const length = partial.readUInt32BE(4);
+          const data = partial.subarray(8, 8 + length);
+
+          if (data.length < length) {
+            break decoder;
```

Biome (1.9.4)

[error] Unnecessary label.

Safe fix: Remove the unnecessary label.
You can achieve the same result without the label.


(lint/complexity/noUselessLabel)